### PR TITLE
Integrate with Travis CI

### PR DESCRIPTION
**Pull request by matthewrmshin**
Run `fcm test-battery` on Travis CI. For some reason, the sort order on the Travis CI box is very different. I have also added some extra polling to ensure the stability of  `t/svn-hooks/03-post-commit-bg.t`.

### CODE REVIEW

**Comment by matthewrmshin** [comment]
@benfitzpatrick Do you mind doing a quick sanity check? I won't spend too much time on it, given that it does not really impact on users.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    - cat >"${HOME}/.bashrc" <<<"export PATH=${PWD}/bin:\$PATH"
+    - source "${HOME}/.bashrc"
+
+install: 
+    - sudo apt-get install -y build-essential gfortran
+    - sudo apt-get install -y libxml-parser-perl libconfig-inifiles-perl
+    # For some reason XML::Parser needs to be installed this way
+    - cpanm 'Config::IniFiles' 'XML::Parser'
+    # Latest Subversion
+    - sudo sh -c 'echo "deb http://opensource.wandisco.com/ubuntu `lsb_release -cs` svn19" >> /etc/apt/sources.list.d/subversion19.list'
```
Comment (benfitzpatrick) [inline .travis.yml]: Are you pulling in Subversion 1.9? If so, how are you getting around #233?

Comment (matthewrmshin) [inline .travis.yml]: According to the installation section of the Travis CI build, it says 1.9.4. No idea how it works.

Comment (benfitzpatrick) [inline .travis.yml]: (We found out that it is skipping Subversion tests when the version is not 1.8)

**Pull request merged**